sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java

```diff
@@ -150,4 +150,14 @@ public void extractKeyPhrases() {
         System.out.println("Extracted phrases:");
         textAnalyticsClient.extractKeyPhrases(document).forEach(keyPhrase -> System.out.printf("%s.%n", keyPhrase));
     }
+
+    /**
+     * Code snippet for recognizing Personally Identifiable Information entity in a document.
+     */
+    public void recognizePiiEntity() {
+        String document = "My SSN is 555-55-5555";
+        textAnalyticsClient.recognizePiiEntities(document).forEach(piiEntity ->
+            System.out.printf("Recognized Personally Identifiable Information entity: %s, category: %s, subCategory: %s, score: %f.%n",
```

Comment:
as we are including offset and length again. should we print that too?

Reply:
offset and length will be include in samples but not on codesnippet and reamde samples